.env_SAMPLE

```diff
@@ -27,6 +27,15 @@ export MYSQL_USER=root
 #export MYSQL_HOST=<hostname>
 #export MYSQL_PORT=<portnumber>
 
+#############################################
+# AWS S3
+#############################################
+#export S3_ENABLED=<boolean_s3_enabled>
```

Comment:
This is awesome to include the datatype in the hint. We should do this elsewhere if it's not clear in other values.